po/keys.pot

```diff
@@ -17,124 +17,6 @@ msgstr ""
 "Content-Type: text/plain; charset=CHARSET\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-#: common/src/main/java/org/candlepin/common/exceptions/mappers/BadRequestExceptionMapper.java:72
```

Comment:
something to note here: The `master` component's configuration in Weblate will have to be changed to find out where the .po files live now (since common/ no longer exists)

Reply:
Noted. 
I'll keep this comment open. Until changes are made.